### PR TITLE
DT-2123: 🗃️ Update list of prisons / gp practices

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +38,44 @@ rather than calling the `/health` endpoint.
 - `/health`: provides information about the application health and its dependencies.  This should only be used
 by prison-register health monitoring (e.g. pager duty) and not other systems who wish to find out the state of prison-register.
 - `/info`: provides information about the version of deployed application.
+
+### Updating prison information
```
Comment (petergphillips) [inline README.md:42]: copied from old prison estate project

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/GpResource.kt [hunk 2]
```diff
@@ -42,7 +42,7 @@ class GpResource(private val prisonService: PrisonService) {
   fun getPrisonFromId(
     @Parameter(description = "Prison ID", example = "MDI")
     @PathVariable @Size(min = 2, max = 12) prisonId: String
-  ): GpDto = GpDto(prisonService.findById(prisonId))
+  ): GpDto = GpDto(prisonService.findById(prisonId.uppercase()))
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/GpResource.kt:45]: also apis should be lenient, so should allow lowercase too

File: src/main/resources/db/migration/V1_4__modify_prisons_gp_practices.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO prison VALUES ('FYI', 'Feltham A (HMPYOI)', true);
```
Comment (petergphillips) [inline src/main/resources/db/migration/V1_4__modify_prisons_gp_practices.sql:1]: checked nomis and found new prison there so added

File: src/main/resources/db/migration/V1_4__modify_prisons_gp_practices.sql [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+UPDATE prison set name = 'Hewell (HMP)' where prison_id = 'HEI';
```
Comment (petergphillips) [inline src/main/resources/db/migration/V1_4__modify_prisons_gp_practices.sql:3]: was _HMP Hewell_ and different from nomis so corrected

File: src/main/resources/db/migration/V1_4__modify_prisons_gp_practices.sql [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+UPDATE prison set name = 'Medway (STC)' where prison_id = 'MWI';
```
Comment (petergphillips) [inline src/main/resources/db/migration/V1_4__modify_prisons_gp_practices.sql:4]: _STC_ lowercase in prison register

File: src/main/resources/db/migration/V1_4__modify_prisons_gp_practices.sql [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+
+INSERT INTO prison_gp_practice VALUES ('KTI', 'Y03175');
```
Comment (petergphillips) [inline src/main/resources/db/migration/V1_4__modify_prisons_gp_practices.sql:6]: grabbed from the _epraccur_ file

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**